cmd/tk/main.go

```diff
@@ -11,7 +11,7 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
-var VERSION = "0.0.1"
+var VERSION = "0.0.1-alpha.1"
```

Comment:
Might be an idea to build with `-ldflags "-X main.VERSION=``git <intelligent way to get tag>``”` for future releases.